config.go

```diff
@@ -219,6 +225,44 @@ func loadConfig() (*config, error) {
 		return nil, err
 	}
 
+	// Setup dial and DNS resolution functions depending on the specified
+	// options. The default is to use the standard golang "net" package
+	// functions. When Tor's proxy is specified, the dial function is set to
+	// the proxy specific dial function and the DNS resolution functions use
+	// Tor.
+	cfg.net = &torsvc.MultiNet{Tor: false}
```

Comment:
The `Tor` option is never set to _true_ below if the config params are well formed.